resources/design-document.md

```diff
@@ -0,0 +1,244 @@
+# Design Document
+
+## Instructions
+
+## Brecghan's Training Matrix(BTM) Design
+
+## 1. Problem Statement
+
+This service is designed to manage a training system for a company's personnel. It allows them to track active employees, what training they have received, what testing they have taken and the results of the test, and if any trainings are set to expire.
+
+This design document describes the service that will provide the training matrix functionality to meet our customers' needs. It is designed to contain employees, trainings, and a testing tracker. It will be able to return current employees, trainings that have been conducted, and tests results from the different trainings.
+
+## 2. Use Cases
+
+U1. As a BTM customer, I want to create an employee
+
+U2. As a BTM customer, I want to be able to create a training
+
+U3. As a BTM customer, I want to be able to create a training series
+
+U4. As a BTM customer, I want to be able to create tests for trainings
+
+U5. As a BTM customer, I want to be able to view all active trainings that are close to expiration
+
+U6. As a BTM customer, I want to be able to update test results for retests
+
+U7. As a BTM customer, I want to be able to view all trainings done within a series
+
+U8. As a BTM customer, I want to be able to view specific trainings
+
+U9. As a BTM customer, I want to be able to view all employees on a specific team
+
+U10. As a BTM customer, I want to be able to view all tests taken by an employee
+
+U11. As a BTM customer, I want to be able to deactivate employees that have left
+
+
+Stretch goals
+
+S1. As a BTM customer, I want to have my trainings auto-update their status based off of their date and months to expiration
+
+S2. As a BTM customer, I want to be able to make the next training in a training series by copying the information from the most recent training
+
+
+## 3. Project Scope
+
+### 3.1. In Scope
+
+* Creating, retrieving, and updating an employee
+* Test list that can be searched by training or employee name
+* Recipes can be retrieved and individual recipes added to meal plan
```

Comment:
yum sounds delicious 🙂   actually sounds like something else belongs here?

Reply:
Yup, that's what I get for copy and pasting, forgot to finish updating this one